.circleci/config.yml

```diff
@@ -90,17 +88,17 @@ jobs:
       - image: ghcci/x86_64-linux:0.0.1
     environment:
       <<: *buildenv
+      GHC_COLLECTOR_FLAVOR: x86_64-linux
     steps:
       - checkout
       - *prepare
       - *submodules
       - *boot
       - *configure_unix
       - *make
-      - *test
       - *bindist
-      - *collectartifacts
       - *storeartifacts
+      - *test
```

Comment:
It would be nice if we could record the testsuite results alongside the artifact for posterity. That being said, this is fine for now.